### PR TITLE
Terraform Provider v2 - Cassandra support

### PR DESCRIPTION
**Pull request by arjunrajinstaclustr**
Support for getting, creating and deleting Cassandra clusters for the Instaclustr Terraform Provider v2.
This is pretty much just a documentation update.
Internal reference: INS-17903

### CODE REVIEW

File: docs/data-sources/cassandra_cluster_v2_instance.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+- **data_centre** (Block List) List of datacentre settings. (see [below for nested schema](#nestedblock--data_centre))
+- **default_user_password** (String) Password of the default user created for the Cassandra Cluster.
+- **lucene_enabled** (Boolean) Adds Apache Lucene to the Cassandra cluster.
+- **name** (String) Name of the cluster.
+- **password_and_user_auth** (Boolean) Enables Password Authentication and User Authorization.
+- **pci_compliance_mode** (Boolean) Creates a PCI compliant cluster, see [PCI Compliance](https://www.instaclustr.com/support/documentation/useful-information/pci-compliance/).
+- **private_network_cluster** (Boolean) Creates the cluster with private network only, see [Private Network Clusters](https://www.instaclustr.com/support/documentation/useful-information/private-network-clusters/).
+- **sla_tier** (String) SLA Tier of the cluster. Non-production clusters may receive lower priority support and reduced SLAs. Production tier is not available when using Developer class nodes. You can find more information [here](https://www.instaclustr.com/support/documentation/useful-information/sla-tier/).
```
Comment (Mitchel-Stig-Instaclustr) [inline docs/data-sources/cassandra_cluster_v2_instance.md:32]: a here link?

File: docs/data-sources/cassandra_cluster_v2_instance.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+### Required
+
+- **id** (String) The ID of this resource.
+
+### Optional
+
+- **cassandra_version** (String) Version of Cassandra to run on the cluster.
+- **data_centre** (Block List) List of datacentre settings. (see [below for nested schema](#nestedblock--data_centre))
+- **default_user_password** (String) Password of the default user created for the Cassandra Cluster.
```
Comment (Mitchel-Stig-Instaclustr) [inline docs/data-sources/cassandra_cluster_v2_instance.md:26]: capitalize cluster here but not below?

File: docs/data-sources/cassandra_cluster_v2_instance.md (hunk 1 above)
Comment (arjunrajinstaclustr) [reply docs/data-sources/cassandra_cluster_v2_instance.md:32]: Copy pasted from existing documentation, will fix

File: docs/data-sources/cassandra_cluster_v2_instance.md (hunk 2 above)
Comment (arjunrajinstaclustr) [reply docs/data-sources/cassandra_cluster_v2_instance.md]: Will fix

**Review by Mitchel-Stig-Instaclustr — APPROVED** [review]

**Review by michael-carter-instaclustr — APPROVED** [review]

**Pull request merged**